cli.js

```diff
@@ -58,6 +61,34 @@ function log(report) {
 	process.exit(report.errorCount === 0 ? 0 : 1);
 }
 
+function open(report) {
+	if (report.errorCount === 0) {
+		return;
+	}
+
+	var files = report.results
+		.filter(function (file) {
+			return file.errorCount > 0;
+		})
+		.map(function (file) {
+			return file.filePath;
+		});
+
+	var editor = process.env.EDITOR;
+
+	if (!editor) {
+		console.log();
+		console.log('`open` option was used, but your $EDITOR environment variable is empty.');
+		console.log('Fix it by setting path to your editor of choice in ~/.bashrc or ~/.zshrc:');
+		console.log();
+		console.log('    export EDITOR=atom');
+		console.log();
+		return;
+	}
+
```

Comment:
`wstorm:line` works,
`wstorm:line:column` does not.

`xo --open` only provides line:column for sublime and atom (and it would not work just to add `wstorm` to the array).